lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -208,5 +208,13 @@ def charged_accomodation_adjustment(charge, number_of_nights)
     def self.historical_minimum_wage_data
       @historical_minimum_wage_data ||= YAML.load_file(Rails.root.join("lib/data/minimum_wage_data.yml"))[:minimum_wage_data]
     end
+
+    def date_from_environment_variable
```

Comment:
@robinjam I would update the minimum wage calculator to get its rates from RatesQuery, just like the other calculators.

This will make life simpler because there will be less duplication (rates files are only read in one place) and we wouldn't need to re-implement the date_from_environment_variable function.

Can you create an initial commit that changes the minimum wage calculators to use RatesQuery, and then make the new rates changes after that commit?

The married-couples-allowance is a good example of how the RatesQuery is used.